backend/src/main/java/com/google/sps/data/MatchPreference.java

```diff
@@ -41,4 +41,18 @@ public static MatchPreference forIntValue(int value) {
     }
     throw new IllegalStateException("Unknown enum value.");
   }
+
+  /** Parse match preference string to MatchPreference */
+  public static MatchPreference forStringValue(String value) {
+    switch (value.toLowerCase()) {
```

Comment:
NIt : do you think the lower case value should be made into a variable so that the function doesn't have to be called each time